src/lib.rs

```diff
@@ -343,6 +343,21 @@ where
             }
         }
     }
+
+    pub fn estimate_fee(&self, confirmation_target: ConfirmationTarget) -> Result<u32, Error> {
+        let client = self.client.lock().unwrap();
+
+        let target_blocks = match confirmation_target {
+            ConfirmationTarget::Background => 6,
+            ConfirmationTarget::Normal => 3,
+            ConfirmationTarget::HighPriority => 1,
+        };
+
+        let estimate = client.estimate_fee(target_blocks).unwrap_or_default();
```

Comment:
I'm not sure if `0` is a valid fee , i.e. if unwrap fails. I would default to `1`.